package.json

```diff
@@ -51,7 +51,7 @@
     "istanbul": "^0.4.4",
     "json-loader": "^0.5.4",
     "mocha": "^2.3.4",
-    "opentracing": "^0.11.0",
+    "opentracing": "^0.13.0",
```

Comment:
I'd suggest locking this to a specific version: i.e., remove the caret and make it `"opentracing": "0.13.0"`.